tests/test_oauth2.py

```diff
@@ -0,0 +1,101 @@
+import logging
+
+import requests
+import responses
+
+logging.basicConfig(level=logging.DEBUG)
```

Comment:
I don't think this is appropriate or useful for a test file.

Reply:
Done, dropped logging